kong/core/balancer.lua

```diff
@@ -173,7 +173,7 @@ local get_balancer = function(target)
   if not balancer then
     -- no balancer yet (or invalidated) so create a new one
     balancer, err = ring_balancer.new({
-        wheelsize = upstream.slots,
+        wheelSize = upstream.slots,
```

Comment:
It seems like this value isn't used by the current version of the balancer lib:

```lua
  if (not opts.wheelSize) and opts.order then
    opts.wheelSize = #opts.order
  end
  if opts.order then
    assert(opts.order and (opts.wheelSize == #opts.order), "mismatch between size of 'order' and 'wheelSize'")
  end
```

From the above snippet, we can see that if `wheelSize` isn't given (which is the case prior to this change), then it is inferred from `opts.order`, which is given. Hence, while this fix is a good catch, it isn't necessary for 0.11.0 or 0.10.4 stable releases (in fact, it is slightly riskier to include it in those).